MaterialComponentsBeta.podspec

```diff
@@ -68,13 +68,13 @@ Pod::Spec.new do |mdc|
 
   mdc.subspec "BottomNavigation" do |component|
     component.ios.deployment_target = '8.0'
-    component.public_header_files = "components/#{component.base_name}/src/beta/*.h"
-    component.source_files = "components/#{component.base_name}/src/beta/*.{h,m}"
+    component.public_header_files = "components/#{component.base_name}/src/MDCBottomNavigationBarController.h", "components/#{component.base_name}/src/MaterialBottomNavigationBeta.h"
```

Comment:
This is a nit pick, but I feel like we should move towards this style:
```
    component.public_header_files = [
        "components/#{component.base_name}/src/MDCBottomNavigationBarController.h",
        "components/#{component.base_name}/src/MaterialBottomNavigationBeta.h",
   ]
```

Reply:
I'd like to do that in a follow-up PR if you're OK with it. I can reformat the whole file.

Reply:
https://github.com/material-components/material-components-ios/pull/6503